### DISCUSSION TITLE
屏蔽b站搜索热榜

### DISCUSSION THREAD

**Discussion by eva-unit-01** [the1812/Bilibili-Evolved#3744] [功能建议 / Ideas]
不知道有没有人有屏蔽b站搜索热榜的需求，现在的热搜越来越莫名其妙了，经常出现一些完全不火，也没什么营养的话题。

希望能添加一个插件，能够屏蔽搜索栏点击之后[https://files.catbox.moe/nrs98w.png](url)，以及在[https://search.bilibili.com/all](url)页面的搜索热榜(`_っ´)

**Comment by Geeyun-JY3** [comment]
（一年多前的预言：https://github.com/the1812/Bilibili-Evolved/issues/2136#issuecomment-900789124

**Reply by eva-unit-01** [reply]
b站现在热搜就离谱，大数据喂屎了属于是，经常一大堆明明没热度的话题，却能搞到热搜，比如这个[搜索结果](https://files.catbox.moe/1cv1xh.png)，所有相关的视频全是几个账号不约而同一两个小时之内发的，内容也差不多，最高播放也才7000，就能到热搜第三，很难怀疑不是收了钱(`_っ´)

**Reply by Geeyun-JY3** [reply]
还是挺多这些跟着热搜改标题的万象天引的机器人号的。

**Comment by hinata2341** [comment]
我在用一个插件，但是它只能屏蔽掉视频播放界面的热搜，功能比较鸡肋

**Comment by Elivis-AI** [comment]
> 不知道有没有人有屏蔽b站搜索热榜的需求，现在的热搜越来越莫名其妙了，经常出现一些完全不火，也没什么营养的话题。
> 
> 希望能添加一个插件，能够屏蔽搜索栏点击之后[https://files.catbox.moe/nrs98w.png](url)，以及在[https://search.bilibili.com/all](url)页面的搜索热榜(`_っ´)

我个人使用的是ADguard插件，然后自己写了一条过滤规则，目前效果很好。
使用方法：在用户过滤器中添加这样一条规则：`||api.bilibili.*/x/web-interface/search/square`。
注：我自己的需求是去除`https://search.bilibili.com/all`中的热榜。

**Reply by funnyjacy** [reply]
这个可以的，nbnb

**Reply by arimakana123** [reply]
现在不可以了吗？ADguard使用失败？

**Reply by arimakana123** [reply]
查了一下最新的API，似乎是由于API更改？我没有网页编程方面的知识。<https://github.com/SocialSisterYi/bilibili-API-collect/blob/master/docs/search/hot.md>   给出了一些变化。
似乎写成||api.bilibili.*/x/web-interface/wbi/search/square就可以了。
